### DISCUSSION TITLE
Error: A connection cannot be made. Ensure that the server is running

### DISCUSSION THREAD

**Discussion by jobosol** [TabularEditor/TabularEditor3#1166] [🙋♀️TE3 Q&A] [answered]
I was updated to Windows 11 Enterprise 23H2. After that I cannot connect to any Power BI Models in Power BI Premium or Azure Analysis Models. 

Every time I try to connect, I get the Microsoft sign in screen and the signing goes but fine, but instead of opening the model I get this error. "A connection cannot be made. Ensure that the server is running."

![image](https://github.com/TabularEditor/TabularEditor3/assets/38816188/3d5ba46e-6262-4c35-9c8a-d73f3262a5d3)

I have tried using "Username and Password" and tried logging in with a serviceprincipal. 
I have tried uninstalling TE3 and deleting the content in AppData\Local\TabularEditor3
I have tried installing version 3.8.0 and 3.10.1 but I get the same error on these versions. 
I have tried setting proxy to "System" in preferences. 
I can connect to the models using TE2 and SMSS. 

I'm guessing something went wrong with the update, but I can't find a way to fix it. Does anyone have a suggestion for what I should do next to get TE3 working again?

**Comment by otykier** [comment]
Does reinstalling the **.NET Desktop Runtime** help?

https://dotnet.microsoft.com/en-us/download/dotnet/6.0

**Reply by jobosol** [reply]
Thanks for the quick suggestion, but I am still getting the same error after a reinstall and reboot. 

The only servers that I can connect to is my localhost analysis services and local power bi desktop models.

**Reply by otykier** [reply]
Sounds like a proxy issue, then. Could you try to change the proxy settings in Tabular Editor 3, under **Tools > Preferences > Proxy settings**. Try to switch between Proxy Type "System" and "None", and vice versa. Restart the application between each change.

**Reply by jobosol** [reply]
Tried. Still getting this 
![image](https://github.com/TabularEditor/TabularEditor3/assets/38816188/263a4b48-6a7f-40fd-971b-89c4f0a9f818)

If I put in an invalid Power BI Premium Workspace I get this. So I guess it can distinguish between real workspaces and fake ones. 

![image](https://github.com/TabularEditor/TabularEditor3/assets/38816188/e834fd3a-3173-4dd5-b15b-ce9983e589f4)

Guess I should head over to the IT department, but do you have any tips I can give them?

**Reply by jobosol** [reply]
I can connect to local onprem SSAS models as well. Regarding proxy TE2 is set to "System" and working, but setting TE3 to "System" does not change anything. 

I see now that after the Microsoft logging screen I get brief glimpse of a windows that says "Object moved" 

First this 
![image](https://github.com/TabularEditor/TabularEditor3/assets/38816188/aedf35ed-d051-4a1a-bb29-b487c4c32a7c)

Then 
![image](https://github.com/TabularEditor/TabularEditor3/assets/38816188/b75bd4d5-ffa0-459f-8a96-9783261cd1f8)

and then it throws the error

![image](https://github.com/TabularEditor/TabularEditor3/assets/38816188/829d9a2e-b069-4aa6-a333-7c8aed7d39a8)

**Reply by otykier** [reply]
Hi @jobosol

That's weird. One last thing you could try is to add `;Protocol Format=xml` to the [connection string](https://learn.microsoft.com/en-us/analysis-services/instances/connection-string-properties-analysis-services?view=asallproducts-allversions#protocol-format).

If this still doesn't solve the issue, then it gets a bit challenging.

The error you're seeing is raised directly from the [AMO/TOM client library](https://www.nuget.org/packages/Microsoft.AnalysisServices.retail.amd64/#versions-body-tab) provided by Microsoft. This is the library that we use for any kind of connectivity with Analysis Services / Power BI, so if this is not working, there's not much we can do about it from our side. We've had some issues with the client library in the past, specifically around proxy server handling in Tabular Editor 3. This is because TE3 uses the .NET Core version of the library, whereas Tabular Editor 2 uses the .NET Framework version.

I see that there's a new version of this library that was released just a few hours ago - if the Protocol Format workaround above doesn't solve the issue, we can try to send you a private build of TE3, that uses the newest version of the library. If this doesn't work either, we'll have to reach out to the Microsoft engineers for further troubleshooting.

**Reply by jobosol** [reply]
Tried and test, but the same result. 

I have done another test on my colleague's PC which had a clean install of Windows 11, and she is getting the same error, so it feels more like something else then an update going wrong. 

So if I can test a private build of TE3 it would be great.

**Reply by otykier** [reply, marked as answer]
Continuing discussion offline.
